pandas/io/json/_normalize.py

```diff
@@ -531,7 +531,14 @@ def _recursive_extract(data, path, seen_meta, level=0):
             raise ValueError(
                 f"Conflicting metadata name {k}, need distinguishing prefix "
             )
-        result[k] = np.array(v, dtype=object).repeat(lengths)
+        if v and isinstance(v[0], abc.Iterable):
```

Comment:
```suggestion
        if v and is_list_like(v[0]):
```

`abc.Iterable` I think would catch strings which are scalars here

Reply:
That's right, thanks!